enterprise/views.py

```diff
@@ -152,14 +152,25 @@ def get_course_specific_consent(self, request, course_id):
         except HttpClientError:
             raise Http404
         next_url = request.GET.get('next')
+        enrollment_deferred = request.GET.get('enrollment_deferred')
+
+        if enrollment_deferred is None:
+            customer = get_object_or_404(
```

Comment:
We don't really have much in the docstring about the error states -- it would be good to include something about the possible 404(s) that can be thrown.